apps/src/utils.js

```diff
@@ -289,6 +289,7 @@ export function getPegasusHost() {
       var name = window.location.hostname.split('.')[0];
       switch (name) {
         case 'localhost':
+        case 'localhost-studio':
```

Comment:
We also have window.dashboard.CODE_ORG_URL which I think tries to do the same thing as this?